### ISSUE TITLE
[BUG REPORT] Going up sneaking in TKR bug

### ISSUE THREAD

**Issue by Skorlex** [CecerMCProjects/NoChangeTheGame#19]
When sneaking in TKR, you go up, there's apparently no way to change a setting to directly disable this issue

- https://youtu.be/rNWRIKJ2cjI

**Comment by Cecer** [comment]
I suspect much of this is server side. The fact that TKR works at all is honestly wizardry. I'll look into it when I get time but I suspect it would be a real mess of logic to solve it client side.

**Comment by Cecer** [comment]
Oh wait... are you saying this only happens with NoChangeTheGame?

**Comment by Skorlex** [comment]
Yeah, that's apparently only a thing that happens with NoChangeTheGame

**Comment by Cecer** [comment]
That's really odd. I'll look into it
